tasks/rss-mentor-dashboard.md

```diff
@@ -1,77 +1,83 @@
+    - grey color - tasks, which should not be started yet (status - no status / todo)
 
-Выбор ментора должен происходить из всех менторов набора, которые содержатся в отдельном excel файле, используя контрол автокомплита (https://bestofjs.org/projects/react-select или https://bestofjs.org/projects/chosen). Выбранный ментор должен сохраняться в localStorage и подставляться при следующей загрузке страницы.
+Selection of mentor should be done from all mentors of the current stream,
+which are contained in separated excel file,
+using autocomplete (https://bestofjs.org/projects/react-select or https://bestofjs.org/projects/chosen).
+Selected mentor should be saved in localStorage and used on next page load.
 
-### Приблизительный внешний вид
+
+### Approximate UI
 ![rss-mentor-dashboard](https://docs.google.com/drawings/d/e/2PACX-1vSV8wUHrWdVDOxyJGl86gC4s_Vmri8Xor3gMRKlB_gF_krSa7UJuydBZ6JMNSpHlQOjW08CtyOqGNny/pub?w=960&h=720)
 
 
-### Ограничения
-  - Должно работать в Chrome
-  - Можно гуглить, общаться между собой и пользоваться stackoverflow
-  - можно использовать lodash
-  - задание должно использовать React
-    
-### Как делать / сдавать
-  - написать nodejs скрипт который
-    - обработает данные с помощью библиотеки [node-xlsx](https://github.com/mgcrea/node-xlsx) из [папки с данными](https://drive.google.com/open?id=1ULj8KjnNNCgUdGunQ1TY00dNbCsqAsHW) . Можно использовать любую другую библиотеку для чтения excel файлов.
-    - будет использовать файлы для обработки библиотекой - Tasks, Mentor score, Mentor-students pairs (информация по парам ментор - студент + информация по соотвествию имени / фамилии ментора его гитхаб аккаунту ) 
-    - исправит возникающие ошибки в данных (неправильное написание гитхаб аккаунтов, неправильные пары ментор-студент). Надо самому решать что исправлять скриптом, а что руками в исходных файлах. Если правятся исходные файлы - надо указать в readme при сдаче таска
-    - соединит (смержит, заджоинит (join)) данные из [google drive папки](https://drive.google.com/open?id=1ULj8KjnNNCgUdGunQ1TY00dNbCsqAsHW)
-      - по студентам ментора
-      - по выставленным оценкам
-    - получит json файл с нужными данными
-    - загрузит данные на html страницу
-  - с помощью react и react component нарисовать дашборд, используя получившийся json
-  - сбилдить веб паком и выложить сбилженные файлы на github pages 
-  - pull request должен содержать 
-    - nodejs script, инструкцию как им пользоваться, 
-    - получившийся json
-    - js файл с реакт компонентой
-    - html страницу, которая подключает реакт, js файл с реактом, загружает json файл с данными, которые отображает react компонента 
-  - запостить ссылку на гитхаб страницу + PR [в форму](https://docs.google.com/forms/d/e/1FAIpQLSctLqyRXrpdgC1vybGjGZoiYwZUUAf6_hSD-gPTZMORS_E1kQ/viewform)
-    
+### Requirements
+  - Should work in Chrome
+  - You can Google, communicate, chat and use Stackoverflow
+  - You may use Lodash
+  - Task should use React
```

Comment:
You **must** use React to complete the task

Reply:
updated, `must` word is bold